### DISCUSSION TITLE
Regarding ComfyUI version shown by the Manager

### DISCUSSION THREAD

**Discussion by andreszs** [Comfy-Org/ComfyUI-Manager#1384] [Q&A] [answered]
Now that the ComfyUI Frontend is independent from ComfyUI itself, it means the user has 2 different repo versions for ComfyUI as an overall piece of software.

Probably the main **Manager** dialog should be cleared up by simply displaying the [ComfyUI](https://github.com/comfyanonymous/ComfyUI) and [ComfyUI Frontend](https://github.com/Comfy-Org/ComfyUI_frontend/tree/main) versions installed vs. latest instead of the changes log.

![image](https://github.com/user-attachments/assets/a1dcfc12-1601-411f-8127-e7459901f080)

I believe the **Manager** window is not the ideal place to display the current changelog. Instead, you could provide a link to the official changelog files (if available) or the repositories directly. This would make the dialog less cluttered and more functional. Honestly, less than 1% of users would scroll through to find the installed ComfyUI version. Most of us simply want to compare the installed version with the latest at a glance, not read the changelog in the Manager dialog, which is a rather cumbersome way of staying updated.

Last, with the current dialog, there's no knowing if we are using an up-to-date or outdated version of the frontend.

**Comment by ltdrdata** [comment, marked as answer]
That dialog is intended more for conveying important information rather than simply displaying the changelog.

With the recent periodic releases, the changelog is included as part of introducing new features in the latest release.

On the contrary, version information is now shown on the About page, so it is unnecessary.
![image](https://github.com/user-attachments/assets/77e06b76-9f57-4949-a1a5-66ab5244a44f)
